static/app/views/dashboards/widgets/widgetLayout/widgetLayout.tsx

```diff
@@ -70,22 +72,23 @@ const Frame = styled('div')<{forceShowActions?: boolean; height?: number}>`
 
   background: ${p => p.theme.background};
 
-  :hover {
-    background-color: ${p => p.theme.surface200};
-    transition:
-      background-color 100ms linear,
-      box-shadow 100ms linear;
-    box-shadow: ${p => p.theme.dropShadowLight};
-  }
-
   ${p =>
-    !p.forceShowActions &&
-    `&:not(:hover):not(:focus-within) {
-    ${TitleHoverItems} {
-      opacity: 0;
-      ${p.theme.visuallyHidden}
-    }
-  }`}
+    p.revealActions === 'hover' &&
+    ` :hover {
```

Comment:
```suggestion
    css` :hover {
```
I had to ask @scttcper bc I couldnt recall what breaks, but not having css will mean that this block isnt covered by stylelint

Reply:
🤦🏻 what a save! I was _wondering_ why the linting isn't working, thanks!